core/src/avm1/test_utils.rs

```diff
@@ -31,8 +31,9 @@ where
 }
 
 macro_rules! test_method {
-    ( $test: ident, $name: expr, $object: expr, $($versions: expr => { $([$($arg: expr),*] => $out: expr),* }),* ) => {
+    ( $test: ident, $name: expr, $object: expr, $($versions: expr => { $($(@epsilon($epsilon: literal))? [$($arg: expr),*] => $out: expr ),* } ),* ) => {
```

Comment:
```suggestion
    ( $test: ident, $name: expr, $object: expr, $($versions: expr => { $($(@epsilon($epsilon: expr))? [$($arg: expr),*] => $out: expr ),* } ),* ) => {
```

`epsilon` doesn't have to be a literal, someone might use an expression here